app/views/holding.njk

```diff
@@ -0,0 +1,49 @@
+{% from 'components/general/components/bc-header/macro.njk' import bcHeader %}
+{% from 'components/general/components/bc-footer/macro.njk' import bcFooter %}
+{% extends './includes/layout.njk' %}
+
+{% block header %}
+  {{ bcHeader({
+      classes: "nhsuk-header__container",
+      showBetaBanner: true,
+      termsHref: "https://forms.office.com/Pages/ResponsePage.aspx?id=Hwf2UP67GkCIA2c3SOYp4nDHKEWnXcFHiqdJhf0fCJtUNDNFRUFZVFU5RkRQTEpWU0RQVlVXMUpRQi4u",
+      logoLink: {
+        href: "https://buyingcatalogue.digital.nhs.uk/",
```

Comment:
Do we definitely want to use an absolute URL here? Will a relative URL work?

Reply:
For the purpose of generating the holding page these URL's need to be absolute since the project is served over localhost.